README.md

```diff
@@ -18,17 +18,17 @@ This documentation is for developers interested in using this Node.js client to
 ## Installation
 
 ```shell
-npm install --save notifications-node-client
+npm install --save @govau-platforms/notify-client
 ```
 
 ## Getting started
 
 ```javascript
-var NotifyClient = require('notifications-node-client').NotifyClient,
-	notifyClient = new NotifyClient(apiKey);
+const { Client } = require('@govau-platforms/notify-client');
```

Comment:
Can we just use `import` syntax? That's the norm now.

Reply:
It's popular but ES modules is still an experimental feature. I'll put a section for typescript and one for javascript like the AWS-SDK.

Reply:
Yeah good call—sorry I was in browser land.